Improvements

5annaha opened the pull request:
* Disable send message if NPC is deceased or killed in action
* Add edit character GM Notes
* Add edit artifact GM Notes
* Remove placeholders and format documents
* Hide extra arrow from sizePerPage change button

5annaha committed:
chore: update package-lock.json

5annaha committed:
feat: disable message sending if sender is deceased

5annaha committed:
fix: remove extra arrow from sizePerPage change button

5annaha committed:
feat: add edit character gm_notes

5annaha committed:
chore: remove console.log

5annaha committed:
chore: remove placeholder for add gm notes during the game

5annaha committed:
chore: remove placeholder for add gm notes during the game

5annaha committed:
chore: format code

5annaha committed:
feat: add edit artifact gm_notes

5annaha committed:
chore: format document

nicou commented on src/api/artifact.js:
```diff
@@ -0,0 +1,18 @@
+
+export async function updateArtifact(artifact) {
+  const requestBody = {
+    ...artifact,
+    gm_notes: artifact.gm_notes?.trim() ?? null,
```
Change to only contain `gm_note`

nicou commented on src/api/character.js:
```diff
@@ -0,0 +1,18 @@
+
+export async function updateCharacter(character) {
+  const requestBody = {
+    ...character,
+    gm_notes: character.gm_notes?.trim() ?? null,
```
Change to only contain gm note

nicou committed:
Improve safety of saving GM notes

Only patch the gm_notes field instead of updating the whole database
row when saving GM notes.
